ui/apps/platform/src/Containers/Clusters/ClusterDeployment.js

```diff
@@ -62,20 +60,15 @@ const ClusterDeploymentPage = ({
                                     />
                                 </div>
                                 <div className="flex justify-center px-3">
-                                    {isDownloadingBundle ? (
-                                        <Spinner isSVG size="lg" />
-                                    ) : (
```

Comment:
Is the conditional rendering no longer needed?

Reply:
Good question. Superseded by props of `Button` element that are illustrated in the last picture.

```tsx
isDisabled={isDownloadingBundle}
isLoading={isDownloadingBundle}
```